packages/server/__snapshots__/5_cdp_spec.ts.js

```diff
@@ -39,7 +39,7 @@ The CDP port requested was 7777.
 Error details:
 
 Error: connect ECONNREFUSED 127.0.0.1:7777
-    [stack trace lines]
+      [stack trace lines]
```

Comment:
im noticing a ton of snapshots that have extra indentations here... is this intended? seems like this shouldn't be the case

Reply:
Yes, this is because the e2e helper now normalizes the whitespace for stack lines. It was too inconsistent between Firefox and Chrome and the various types of errors.